### ISSUE TITLE
Boundary for FRLA

### ISSUE THREAD

**Issue by jakecoolidge** [nationalparkservice/places-data#197]
The boundary we use appears a bit off from what is displayed on the park site:

In Park Tiles it appears to be located off Welch Rd.:
![frla-park-tiles](https://cloud.githubusercontent.com/assets/11392872/10675877/e142cd9e-78bf-11e5-9c43-ef0dc7be1a41.png)

![frla-nps](https://cloud.githubusercontent.com/assets/11392872/10675876/e13505b0-78bf-11e5-9eae-726ce7bde7de.png)
Map above found [here.](http://www.nps.gov/frla/planyourvisit/directions.htm)

**Comment by chadlawlis** [comment]
Strange, it's pulling from Lands so it shouldn't be far off... But you're right, the FRLA map locates it across the street. I can get in touch with Nigel about this.

**Comment by jakecoolidge** [comment]
I was hoping the Park Atlas data we recently acquired had a better representation of this boundary, but I couldn't identify one. Hopefully Nigel has something.

**Comment by chadlawlis** [comment]
From Nigel @ NER, 1/15/16:

> FRLA Boundary: The version in the NPS Lands db has not been updated during Roger Johnson's tenure.   As depicted on Park Tiles, the location is incorrect.  The FRLA site is across Warren Street from that location. I don't have a digital boundary here but am checking with Mark Duffy to see if he can share anything for FRLA and will let you know as soon as I hear from him.  In the 1990s we had an early Autocad version of the FRLA boundary but it was not georef'd.  I _may be able to dig that up if Mark doesn't have anything more helpful.
> 
> OSM has the park placed correctly in the screenshot below.  (The screenshot includes Welch Street just for reference.)  It is a large residential parcel at 99 Warren Street in Brookline, MA and is bounded by Warren, Dudley and Fairmont Streets.  99 Warren is surely the operational address it had when Olmsted lived and worked there and so it remains now - for mail delivery and such.  There is a historic driveway coming west off Warren Street that enters the east side of the property.  It dates from Olmsted's occupancy but is not used by any actual vehicles now.  Staff and visitors in cars use the driveway around the corner which comes south off Dudley street. entering the north side of the property where there is a parking lot.  

![frla](https://cloud.githubusercontent.com/assets/1238636/12570773/da0c88a8-c397-11e5-99ce-1b3d3ad6750b.png)

**Comment by chadlawlis** [comment]
Thanks to the NPMap.js' geocoder we can add a point for this based on the NPS.gov address provided: http://www.nps.gov/frla/contacts.htm

99 Warren Street
Brookline, Ma 02445

Hopped into http://www.nps.gov/lib/npmap.js/3.0.7/examples/geocoder-control.html, searched for the address, and ran `NPMap.config.L.getCenter();` in Chrome's console at landing zoom/extent, providing:

`L.LatLng {lat: 42.325445721117916, lng: -71.13200604915617}`

Plugged this into an update query and we should be good to go with a label point:

``` sql
-- update geom_point with coordinates in 4326
UPDATE parks
SET geom_point = (SELECT ST_GeomFromText('POINT(-71.13200604915617 42.325445721117916)', 4326))
WHERE unit_code = 'frla';

-- set SRID to 3857
UPDATE parks
SET geom_point = ST_Transform(ST_SetSRID(geom_point, 4326), 3857)
WHERE unit_code = 'frla';
```

We still need a `geom_poly` for this, though - we could trace something rudimentary, if need be, but I suggest we stick to the `geom_point` created from the official NPS address and only add a geometry when it comes from Nigel/Mark.

**Comment by chadlawlis** [comment]
From Nigel, 2/26/16:

> Mark Duffy is actively working on sorting things out ... I believe he sent a final happy version of FRLA to Roger Johnson or Jean Olson in WASO Lands either today or yesterday (as in attached - still not official - pdf)

[frlasm101_draft.pdf](https://github.com/nationalparkservice/places-data/files/151455/frlasm101_draft.pdf)

I have removed the current `geom_poly` given it is inaccurate and will upload the new geometry once this is available from Mark/Lands.

**Comment by chadlawlis** [comment]
From Nigel, 3/2/16:

> FRLA is "going through final routing of approval process" (in Philadelphia). Once Rachel approves it will go to Ft. Collins. (this overrides my earlier understanding of the status for FRLA)

Her earlier understanding was Mark had sent a new boundary to WASO Lands on 2/29/16.

**Comment by chadlawlis** [comment]
This will be fixed with the new Lands release 20160331 #697. The newly available boundary looks like this:

![frla_lands_20160331](https://cloud.githubusercontent.com/assets/1238636/14390637/50d8d010-fd75-11e5-9fd2-f2108b672119.png)
